src/client/websocket/handlers/TYPING_START.js

```diff
@@ -1,18 +1,54 @@
 'use strict';
 
 const { Events } = require('../../../util/Constants');
+let timeout = setTimeout(() => null, 10000);
```

Comment:
whats the deal with this? it will start breaking when multiple people type, should definetly refactor that lol

Reply:
Yeah, I overlooked that, sorry. I've updated it now, and it works as intended from my testings.

Reply:
Yeah, I'm dumb. I've patched it and now it *should* technically work as intended. Thank you for the constant support & feedback, it's appreciated greatly :+1: